### PR TITLE
Updated the UI docs for the authresource

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi,

I have tried to update the UI docs for the authresource.

Poornima

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js [hunk 1]
```diff
@@ -34,6 +70,26 @@ function authResource($q, $http, umbRequestHelper, angularHelper) {
...
+    * Verify the two-factor authentication code entered by the user against the provider
+    *
+    * ##usage
+    * <pre>
+    * authResource.verify2FACode(provider, code)
+    *    .then(function(data) {
+    *        //Do stuff ...
+    *    });
+    * </pre>
+    * @param {string} provider Name of the provider
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: @poornimanayar I guess it should be "Name of the provider" here.

Comment (poornimanayar) [reply src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: Hi @bjarnef ,

I am not sure I understood what the comment is? 

Poornima

Comment (bjarnef) [reply src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: You have written "provider Name of the provider". Shouldn't it just be "Name of the provider"?

Comment (poornimanayar) [reply src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: Thanks for clarifying @bjarnef . I am following the examples from other methods and it seems the convention for a parameter is `@param {type} name-of-parameter desription-of-parameter`

Comment (bjarnef) [reply src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: @poornimanayar ahh yes, I think you're are right, that is how `ngdoc` handle the docs in the comments. Just looked wrong when viewing the changes on GitHub 🤦‍♂️

Comment (poornimanayar) [reply src/Umbraco.Web.UI.Client/src/common/resources/auth.resource.js:88]: No worries at all @bjarnef :)

**Review by nathanwoulfe — APPROVED** [review]
Thanks @poornimanayar, ngdocs syntax is weird, but your update is not.

🎅

**Pull request merged**